itest/lnd_psbt_test.go

```diff
@@ -27,13 +28,95 @@ import (
 // by using a Partially Signed Bitcoin Transaction that funds the channel
 // multisig funding output.
 func testPsbtChanFunding(ht *lntest.HarnessTest) {
-	// First, we'll create two new nodes that we'll use to open channels
-	// between for this test. Dave gets some coins that will be used to
-	// fund the PSBT, just to make sure that Carol has an empty wallet.
-	carol := ht.NewNode("carol", nil)
-	dave := ht.NewNode("dave", nil)
+	const burnAddr = "bcrt1qxsnqpdc842lu8c0xlllgvejt6rhy49u6fmpgyz"
+
+	testCases := []struct {
+		name           string
+		commitmentType lnrpc.CommitmentType
+	}{
+		{
```

Comment:
also test for non-anchor?

Reply:
Didn't want to blow up the matrix too much. And since the difference in the PSBT funding process is only really different between taproot chans and everything before them, I don't think adding more channel types adds much value.